### ISSUE TITLE
Raw PDF file is sent and rendered on canvas instead of Editor.bin format

### ISSUE THREAD

**Issue by Maaayank** [ONLYOFFICE/DocumentServer#2515]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Windows Server

### Version information

7.5.0

### Expected Behavior

On Opening a PDF file in DocumentServer, it should be converted to Editor.bin format and then sent to browser for rendering.
Like it used to happen in 7.0.1 version.

### Actual Behavior

 Raw downloadable PDF document is sent to browser for rendering which poses a security risk of accessing file content without permission.

### Reproduction Steps

Open any PDF file in DocumentServer (7.1.0 and beyond)

### Additional information

Setup:
- DocumentServer v7.5.0 (windows deployment)
- Chrome browser

Usecase:
 - To share and display user files with view only permission
 - User cannot copy content and has a default watermark.
 
Is there any way to send Editor.bin instead of raw .pdf files to browser ?

**Comment by igwyd** [comment]
@Maaayank thank you for the report, i created ticket #65515. I cheked that "user cannot copy information by disabling copy permission", but it expected behavior or what do you mean?

**Comment by Maaayank** [comment]
@igwyd  Yes that's the expected behavior I was pointing out to, which are at stake because of raw PDF download
